pyomo/solvers/tests/checks/test_no_solution_behavior.py

```diff
@@ -0,0 +1,110 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright 2017 National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+import os
+import types
+try:
+    import new
+    new_available=True
+except:
+    new_available=False
+
+import pyutilib.th as unittest
+
+from pyomo.solvers.tests.models.base import test_models
+from pyomo.solvers.tests.testcases import test_scenarios
+
+# The test directory
+thisDir = os.path.dirname(os.path.abspath( __file__ ))
+
+# Cleanup Expected Failure Results Files
+_cleanup_expected_failures = True
+
+#
+# A function that function that returns a function that gets
+# added to a test class.
```

Comment:
Can you clarify this?  I feel that there is a typo here, but the logic is sufficiently complex that I'm not sure...

Reply:
It was a typo